lib/almodovar.rb

```diff
@@ -16,14 +16,27 @@
 require 'almodovar/to_xml'
 
 module Almodovar
-
   class << self
+    DEFAULT_SEND_TIMEOUT = 30
+    DEFAULT_CONNECT_TIMEOUT = 30
+    DEFAULT_RECEIVE_TIMEOUT = 30
+
     def default_options
-      @default_options ||= {
-        :timeout => 120,
-        :connect_timeout => 30,
-        :user_agent => "Almodovar/#{Almodovar::VERSION}"
+      default = {
+        send_timeout: DEFAULT_SEND_TIMEOUT,
```

Comment:
> None of the users of the gem have requested it.

I requested in the issue. With the coming work in Bond having this settings, in addition to the exception would be very useful.

As a normal user of the library you don't need to set up timeouts or worry about this, is only in special cases when having these options is useful. Remember that Almodovar is just a wrapper around a http client applying some conventions, without expose nothing in particularly about the http request, being simpler to use but at the same time very limited. Lately I had the idea that we should expose more things to the library user like the timeout exception or the http status code, to make it for useful and flexible in more scenarios. 

I also asked @AlfonsoUceda to investigate that when the exception is raised if is possible to identify for which timeout belongs, which is key for troubleshooting some issues.